metadata-service/war/src/main/webapp/WEB-INF/beans.xml

```diff
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xmlns:context="http://www.springframework.org/schema/context"
-       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.0.xsd http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-3.0.xsd">
+       xmlns:task="http://www.springframework.org/schema/task"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
```

Comment:
Why these changes?

Reply:
To get the scheduled task lib?

Reply:
And do there need to be changed to build.gradle?

Reply:
Yep, these are to get scheduled task functionality as described here: https://howtodoinjava.com/spring-core/spring-scheduled-annotation/ 

AFAIK there aren't any required changes to build.gradle and the scheduled functionality is working. @dexter-mh-lee can you confirm?